doc/source/user/workbench.rst

```diff
@@ -0,0 +1,178 @@
+that facilitates the use of multiple tools in analysis
+by providing a commom user interface for model creation, review, execution, output,
+and visualization for integrated codes. Instructions on how to download and install
+Workbench can be found on the Workbench
+`Gitlab repository <https://code.ornl.gov/neams-workbench/downloads>`.
+
+To set up WATTS in the Workbench environment, user needs to first provide the path to
+where Workbench is installed in `workbench.sh` under the `scripts` directory. User can
+then run `setup_conda_wb.sh` under the same directory to set up WATTS within
+the Workbench environment.
+
+Optional: To install OpenMC in the Workbench environemnt, run `setup_openmc.sh`.
+
+To finish setting up WATTS, open Workbench and go to the `File` tab at the top-left corner
+then select `Configurations` from the drop menu. In the `Application configurations`
+window, click `Add` on the top most row then select `WATTS` from the pop-up window.
+In the list of `Application Options`, input the path of `watts_ui.py` to `Executable`.
+The file should exist by default in `/watts/src/watt_ui/`. Next, click `Load Grammar`.
+Click `OK` to complete the setup.
+
+Optional: Environment variables can be added be added under `Run Environment`.
+
+WATTS Workbench Input File
+++++++++++++++++++++++++++
+
+To create a new WATTS input file in Workbench, go to `File`, select `New File`,
+and select `WATTS files (*.son)` from the drop down menu. An existing `*.son`
+file can also be dragged and dropped to the Workbench window.
+
+The WATTS input file utilizes a hierarchical block-based system where a block
+is defined with opening and closing curly braces `{ }`. A `watts` block is first
+created within which other blocks and variables can be defined. Within the `watts`
+block, `workflow_dir` needs to be defined as the path to the working directory
+(where all templates and extra files are located).
+
+Plugins
+~~~~~~~
+
+The `plugins` block is required. Within the `plugins` blocks are
+`plugin` sub-blocks where WATTS plugins are defined.
+
+.. code-block:: text
+    plugins{
+        plugin(ID1){
+            code = moose
+            template = "sam_template"
+            exec_dir = SAM_DIR
+            exec_name = "sam-opt"
+            extra_inputs=["file1", "file2", "file3"]
+            extra_templat_inputs=["template1", "template2", "template3"]
+            show_stderr = False
+            show_stdout = False
+        }
+
+        plugin(ID2){
+            code = PyARC
+            template = "pyarc_template"
+            executable = "path/to/pyarc/executable"
+            show_stderr = False
+            show_stdout = False
+        }
+    }
+
+Multiple `plugin` sub-blocks can be defined within the `plugins` block where
+each plugin is given a unique identity, as represented by `(ID1)` and `(ID2)`
+in the snippet above. For each sub-block the basic inputs are::
+
+     `code` : Name of the application
```

Comment:
it would be nice to have list of options, if required or optional, etc. In fact, I can show you how such documentation can be automatically generated with Workbench (and maybe added to this documentation)